.github/configuration/upgrade-test-configuration.yml

```diff
@@ -61,7 +61,7 @@ upgrade-version: [{
 {
   upgrade-path: [
     {
-      version: '14.12',
+      version: '14.13',
```

Comment:
Shouldn't we add new path instead of overiding previous one?

Reply:
We don’t want too many github actions unnecessarily, we already have 14.11 -> latest so updating 14.12 -> latest to 14.13 -> latest should be sufficient.